cypress/integration/regressions.spec.js

```diff
@@ -200,4 +200,20 @@ context('Regressions', () => {
     cy.visitSearch('Edmonton')
     cy.get('[data-cy=search-results]').should('be.visible')
   })
+
+
+  /**
+   * Ensure preverbs don't get weird search results
+   *
+   * See: https://github.com/UAlbertaALTLab/cree-intelligent-dictionary/issues/355
+   */
+  it('should not present un-related translation for preverbs', function () {
+    cy.visitSearch('nitawi-')
+
+    // there should be only one result
+    cy.get('[data-cy=search-results]').should('have.length', 1)
+
```

Comment:
So the issue was that the _pop-ups_ were not getting the right definition. "nitawi-" should have a pretty constant definition containing the words "go and", so could you add an assertion for the content as well? Something like:

```suggestion
      .and('should.contain', 'go and')
```